### ISSUE TITLE
Support python3-gpg (for Ubuntu 20.04 LTS)

### ISSUE THREAD

**Issue by bernhardreiter** [Intevation/intelmq-mailgen#44]
GnuPG has published official Python bindings.

The currently used GnuPG python bindings (pygpgme) are being phased out from GNU/Linux distributions and are not very actively maintained.

We shall support the official bindings (and test with Ubuntu 20.04 LTS)
additionally.  The official bindings are in package `python3-gpg` (see https://packages.ubuntu.com/focal/python3-gpg).

Support can be dropped for pygpgme, once Ubuntu 16.04 LTS stops getting maintenance updates (April 2021) https://ubuntu.com/about/release-cycle https://wiki.ubuntu.com/Releases

### details on old library
Is https://launchpad.net/pygpgme (which would be the package `python3-gpgme` for Ubuntu 16.04 LTS (https://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=python3-gpgme&searchon=names)

**Comment by bernhardreiter** [comment]
Dropping support for pygpgme and elder Python version right now, as there is no planned release in the next months and if one would become necessary, if could be done as a new branch from a previous version.

**Comment by bernhardreiter** [comment]
45b20ea520fa810fcaae77d7a2b5ed1128d93af3 ports `test_sign.py` to python3-gpg.
Here is running it in a small VM (with speed diagnostics enabled):
```
ALLTESTS=1 python3 -m unittest discover --start-directory tests -v -p test_si\*
test_sign_nomime (test_sign.SignTestCase) ... ok
test_speed (test_sign.SignTestCase) ... 
Time elapsed for 100 iterations: 3.677
That is 27.2 signatures per second.
ok

```

**Comment by swilde** [comment]
At least `intelmqmail/cb.py' still tries to import the module gpgme which fails for me using 
a installation of latest HEAD using a freshly build debian package.

@bernhardreiter will be looking into this.

**Comment by bernhardreiter** [comment]
Confirming that intelmqmail/cb.py' still needs to be ported.

**Comment by bernhardreiter** [comment]
fixed with 0f4c65092dbcda47f22254356388ad7f712da614
